app/renderer/src/containers/AppContainer.js

```diff
@@ -1,26 +1,30 @@
 import { ipcRenderer } from 'electron';
-import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
 import App from '../components/App';
-import * as actionCreators from '../actions/creators';
 import {
   IPC_WINDOW_RESIZE,
+  IPC_WINDOW_COLLAPSE,
+  IPC_WINDOW_EXPAND,
   IPC_LOAD_THEME,
   IPC_QUERY_COMMAND,
 } from '../../../ipc';
 
-const AppContainer = class extends Component {
+export default class AppContainer extends Component {
   static displayName = 'AppContainer';
 
   state = {
     // the current query value
     query: '',
+    // the current set of results
+    results: [],
     // the current theme
     theme: {},
     // currently pressed keys
     keys: [],
+    // details pane content
+    details: '',
+    // currently selected index
+    selectedIndex: 0,
```

Comment:
the name already clearly states what this variable is. I don't think a comment is necessary.
WDYT?